torchvision/prototype/features/_collate.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Callable, Dict, Optional, Sequence, Tuple, Type, Union
```

Comment:
I've put this into `features` for now, put when everything is rolled out it should probably be in `datasets`.